client/post-editor/editor-ground-control/quick-save-buttons.jsx

```diff
@@ -34,12 +34,13 @@ const QuickSaveButtons = ( {
 	post,
 	translate,
 	onSave,
+	...props
 } ) => {
 	const onSaveButtonClick = () => {
 		onSave();
 		const eventLabel = isPage( post ) ? 'Clicked Save Page Button' : 'Clicked Save Post Button';
-		this.props.recordEditorEvent( eventLabel );
-		this.props.recordEditorStat( 'save_draft_clicked' );
+		props.recordEditorEvent( eventLabel );
+		props.recordEditorStat( 'save_draft_clicked' );
```

Comment:
These work without the `props` prefix too - I wonder if that would be simpler?

Reply:
We would see 'no-shadow' lint errors unfortunately :( - Something I don't personally agree with, since most folk just get around this by renaming functions in the `connect` call.